src/main/resources/application-localhost.properties

```diff
@@ -1,8 +1,9 @@
 # =========================================================================
 # Spring-related.
 spring.config.activate.on-profile=localhost
-spring.config.import=classpath:custom.properties, optional:file:${user.home}/.${user.name}-conf/myiam-overrides.properties
-
+spring.config.import=classpath:custom.properties, optional:file:${user.home}/.${user.name}-conf/uh-groupings-ui-overrides.properties
+# Thymeleaf template caching enabled.
```

Comment:
This comment should be "disabled" instead of "enabled".

Reply:
done, thank you